formula/src/main/java/com/instacart/formula/FormulaRuntime.kt

```diff
@@ -85,75 +86,107 @@ class FormulaRuntime<Input : Any, Output : Any>(
                     effectQueue.addLast(effects)
                 }
 
-                process(isValid)
+                run(isValid)
             }
 
-            val processorManager: FormulaManager<Input, Output> =
-                FormulaManagerImpl(implementation, input, lock, transitionListener)
-
-            manager = processorManager
-
-            process(false)
+            manager = FormulaManagerImpl(implementation, input, transitionListener)
+            forceRun()
             hasInitialFinished = true
 
             lastOutput?.let {
                 onOutput(it)
             }
         } else {
-            process(false)
+            forceRun()
         }
     }
 
+    fun terminate() {
```

Comment:
Moving this into `FormulaRuntime` since it's only used from here (no need to have it on the `FormulaManager` interface).